src/examples/10-10CG/example.js

```diff
@@ -30,7 +30,7 @@ const example = {
     fullName: { first: 'John Doe', middle: 'A', last: 'Doe' },
     ssnOrTin: '787787854',
     dateOfBirth: '1989-11-06',
-    gender: 'U',
+    gender: 'F',
```

Comment:
This example should only have required properties defined. It's used to test the most minimal input a user can provide. So we should remove the gender value/property. This way we know dependent libraries (i.e. vets-api) can function without a gender provided.

Reply:
Removed all instances of `gender` from this example